tests/entrypoints/openai/test_mp_api_server.py

```diff
@@ -6,21 +8,22 @@
 
 
 @pytest.mark.asyncio
-async def test_mp_crash_detection():
-
-    with pytest.raises(RuntimeError) as excinfo:
-        parser = FlexibleArgumentParser(
-            description="vLLM's remote OpenAI server.")
-        parser = make_arg_parser(parser)
-        args = parser.parse_args([])
-        # use an invalid tensor_parallel_size to trigger the
-        # error in the server
-        args.tensor_parallel_size = 65536
-
-        async with build_async_engine_client(args):
-            pass
-    assert "The server process died before responding to the readiness probe"\
-          in str(excinfo.value)
+async def test_mp_crash_detection(capfd):
```

Comment:
I haven't used the `capfd` fixture before, but I was expecting to see it used here to check stderr for the failure log?

Reply:
meant to remove that